packages/@okta/vuepress-site/docs/guides/sign-into-spa/create-okta-application/index.md

```diff
@@ -12,7 +12,7 @@ Before you can sign a user in, you need to create an Okta app integration that r
 1. Pick **Single-Page App (SPA)** as the platform and click **Create**.
     > **Note:** It is important to choose the appropriate application type for apps which are public clients. Failing to do so may result in Okta API endpoints attempting to verify an app's client secret, which public clients are not designed to have, hence breaking the sign-in or sign-out flow.
 1. Enter a name for your app integration (or leave the default value).
-1. Enter values for the **Login redirect URI**. This is the callback from <GuideLink link="../define-callback/">Define a callback route</GuideLink>. Add values for local development (for example, `http://localhost:8080/login/callback`) and production (for example, `https://app.example.com/login/callback`).
+1. Enter values for the **Login redirect URI**. This is the callback from <GuideLink link="../define-callback/">Define a callback route</GuideLink>. Add values for local development (for example, `http://localhost:8080/login/callback` or `http://localhost:8080/authentication/login-callback` for Blazor) and production (for example, `https://app.example.com/login/callback`).
```

Comment:
I wonder if we should change blazor callback to be consistent with other SPAs. Not in the scope now, but something to consider.

Reply:
It's possible and I mentioned somewhere in this Doc how it can be achieved but it requires extra efforts, default Microsoft callback point is authentication/login-callback